src/Tokenizer/Token.php

```diff
@@ -217,6 +222,8 @@ public function getPrototype()
     /**
      * Get token's content.
      *
+     * It shall be used only for getting the content of token, not for checking it against excepted value.
```

Comment:
We do when it comes to reading github diffs...

![image](https://cloud.githubusercontent.com/assets/2829600/26704664/9d6d0204-4728-11e7-93ea-bf65dbb24b53.png)